core/base/document.php

```diff
@@ -1375,4 +1390,59 @@ private function add_handle_revisions_changed_filter() {
 	private function remove_handle_revisions_changed_filter() {
 		remove_filter( 'wp_save_post_revision_post_has_changed', [ $this, 'handle_revisions_changed' ] );
 	}
+
+	/**
+	 * Validate elements settings.
+	 *
+	 * Iterate all elements via the `validate_controls` method.
+	 *
+	 * @param array $elements
+	 *
+	 * @return array
+	 */
+	private function validate_elements_settings( array $elements ) {
```

Comment:
Please make it a general loop so we can use it for other purposes (such as saving assets used by the page)